src/styles/header/style.scss

```diff
@@ -0,0 +1,77 @@
+$c-main-accent: #060b35;
+$c-blue-accent: #00acdc;
+
+header {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
- Apply the main font family to the body or html element
- move to normalize file